src/server/llmQueries.ts

```diff
@@ -49,145 +61,183 @@ type LanguageCode = "nn" | "nb" | "en";
+          Du har fått i oppgåve å lese gjennom og skrive ei oppsummering på NN til artikkelen i <draft> tagget, tittelen til artikkelen er "${title}". 
+          Oppsummeringa skal vere på mindre enn 300 ord og skrives i ein <answer>-tag.
+          ${DEBUG_INSTRUCTION.en}
+          ${ERROR_INSTRUCTION.en}
         </Instructions>`,
 };
 
 type MetaDescriptionQuery = Record<LanguageCode, (text: string, title: string) => string>;
 
 const META_DESCRIPTION_QUERY: MetaDescriptionQuery = {
-  nb: (text: string, title: string) => `<draft>${text}</draft>
+  nb: (text: string, title: string) => `
+        <draft>${text}</draft>
         <Instructions>
           Du har lang erfaring fra utdanningssektoren.
           Du er spesialist i å finne gode måter å beskrive fagartikler på, slik at man fra en kort oppsummering klarer å forstå hva artiklene handler om.
-          Du har fått som oppdrag å lese gjennom artikkelen i <draft> med tittel "${title}" og skrive en metabeskrivelse av artikkelen på NB. Metabeskrivelsen skal være på mindre enn 15 ord.
-          Svaret skal leveres med refleksjoner på hvorfor du har valgt å inkludere de punktene du har valgt først, så selve metabeskrivelsen skrevet i en <answer>-tag.
-          Om du ikke klarer å levere svaret på formatet <answer> som er spesifisert skal første linje i responsen være <ERROR>-tag.
-        </Instructions>`,
-  en: (text: string, title: string) => `<draft>${text}</draft>
+          Du har fått som oppdrag å lese gjennom artikkelen i <draft> tagget med tittel "${title}" og skrive en metabeskrivelse av artikkelen på NB.
+          Metabeskrivelsen skal være på mindre enn 15 ord og skal skrives i en <answer>-tag.
+          ${DEBUG_INSTRUCTION.nb}
+          ${ERROR_INSTRUCTION.nb}
+         </Instructions>`,
+  en: (text: string, title: string) => `
+        <draft>${text}</draft>
         <Instructions>
           You have an extensive experience from the educational sector.
           You are specialized in finding good ways to describe educational articles, in a way that the reader can understand what the article is about from a very short summary.
-          Your task is to read through the article in <draft> with title ${title} and write meta description in EN. The meta description must be in less than 15 words.
-          The response must be given with reflections as to why you have chosen to include the selected points first, then the meta description in an <answer>-tag.
-          If you can not deliver the answer in the given format <answer>, the first line in the response should be <ERROR>-tag.
-        </Instructions>`,
-  nn: (text: string, title: string) => `<draft> ${text} </draft>
+          Your task is to read through the article within the <draft> tag with title ${title} and write meta description in EN.
+          The meta description must be in less than 15 words and written inside a <answer>-tag.
```

Comment:
`The meta description must be less than 15 words, and written inside an <answer> tag`